tests/test_customers.py

```diff
@@ -124,3 +124,36 @@ def test_customer_get_related_payments(client, response):
     customer = client.customers.get(CUSTOMER_ID)
     payments = customer.payments.list()
     assert_list_object(payments, Payment)
+
+
+def test_create_customer_bad_request(client, response):
+    response.post("https://api.mollie.com/v2/customers", "bad_request", status=400)
+
+    with pytest.raises(BadRequestError) as exc:
+        client.customers.create(
+            {
+                "name": "Customer A",
+                "email": "customer@example.org",
+                "locale": "nl_NL",
+            },
+            idempotency_key="test_idempotency_key",
+        )
+
+    assert exc.value.idempotency_key == "test_idempotency_key"
+
+
+def test_update_customer_bad_request(client, response):
```

Comment:
This test should be part of the `test_api_client` file, as they test Idempotency-Key behaviour. It has nothing particular to do with customers.